src/Scripts/Configs.py

```diff
@@ -9,6 +10,17 @@
 
 defaultplayerarray = {"name": "New Player", "steamid": "0", "adminlevel": "0"}
 
+def GetSysLang() -> str:
+    sysDefaultLocale = locale.getdefaultlocale()[0]
+    if sysDefaultLocale.lower() == 'fr-FR'.lower() or sysDefaultLocale.lower() == 'fr-BE'.lower() or sysDefaultLocale.lower() == 'fr-CA'.lower() or sysDefaultLocale.lower() == 'fr-LU'.lower() or sysDefaultLocale.lower() == 'fr-CH'.lower():
+        return 'Fran\u00e7ais'
+    elif sysDefaultLocale.lower() == 'zh_CN'.lower() or sysDefaultLocale.lower() == 'zh-SG'.lower() or sysDefaultLocale.lower() == 'zh-Hans'.lower():
+        return 'SChinese'
+    elif sysDefaultLocale.lower() == 'zh-TW'.lower() or sysDefaultLocale.lower() == 'zh-HK'.lower() or sysDefaultLocale.lower() == 'zh-MO'.lower() or sysDefaultLocale == 'zh-Hant'.lower():
+        return 'TChinese'
+    else:
+        return "English"
+
```

Comment:
an easier and cleaner way to do this is to just get the system locale and only use the first 2 letters then just rename the languages files to 2 letters names `en.json`, `fr.json` etc... 
as for the chinese we can do a simple check like `if ("zh" in locale)  and ("tw" in locale) then return "TChinese"`

Reply:
edited